packages/d3fc-series/src/webgl/repeat.js

```diff
@@ -0,0 +1,52 @@
+import {rebindAll, exclude} from '@d3fc/d3fc-rebind';
```

Comment:
Nitpick: Missing spaces after and before curly brackets

Reply:
Not sure why the linter didn't pick this up